src/hacbs/components/PipelineRunDetailsView/RelatedPipelineRuns.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { Bullseye, Button, Popover, Spinner } from '@patternfly/react-core';
+import { PipelineRunGroupVersionKind } from '../../../shared';
+import { useNamespace } from '../../../utils/namespace-context-utils';
+import { PipelineRunLabel } from '../../consts/pipelinerun';
+import { PipelineRunKind } from '../../types';
+
+const RelatedPipelineRuns = ({ pipelineRun }) => {
+  const namespace = useNamespace();
+
+  const [relatedPipelineRuns, relatedPipelineRunsLoaded] = useK8sWatchResource<PipelineRunKind[]>({
+    groupVersionKind: PipelineRunGroupVersionKind,
+    namespace,
+    isList: true,
+    selector: {
+      matchLabels: {
+        [PipelineRunLabel.COMMIT_LABEL]:
+          pipelineRun.metadata?.labels[PipelineRunLabel.COMMIT_LABEL],
+      },
+    },
+  });
+
+  const loading = (
+    <Bullseye>
+      <Spinner data-test="spinner" />
+    </Bullseye>
```

Comment:
You should use an inline PF Skeleton component instead of rendering this spinner centrally in a bullseye.

Reply:
done, thanks